backend/agents/adapters_test.py

```diff
@@ -0,0 +1,39 @@
+from agents.tool import tool_metadata

```

Comment:
If you add these method names to the `__init__.py` we can import all of this in 1 line
```python
from agents import tool_metadata, get_mistral_tool, Parameter
```

Reply:
like it, changing